src/js/common/dom.js

```diff
@@ -164,6 +164,13 @@ class DOM {
       processed.push('options')
     }
 
+    // disabling all initially selected options because we are setting them manualy at #330
+    if (element.tagName === 'OPTION') {
+      setTimeout(() => {
+        element.selected = false
```

Comment:
what is the reason for using timeout?

Reply:
https://github.com/Draggable/formeo/pull/293#discussion_r551952642